src/providers/doctor/doctor.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { AngularFirestore, AngularFirestoreCollection } from 'angularfire2/firestore';
+import { AlertController } from 'ionic-angular';
+import { Doctor } from '../../models/doctor';
+import { Observable } from 'rxjs/Rx';
+
+@Injectable()
+export class DoctorProvider {
+  private doctorCollection: AngularFirestoreCollection<Doctor>;
+  private doctorObservable: Observable<Doctor[]>;
+
+  constructor(public fireStore: AngularFirestore, public alertCtrl: AlertController) {
+    this.doctorCollection = fireStore.collection<Doctor>('/doctors');
+    this.doctorObservable = this.doctorCollection.snapshotChanges().map((actions) => actions.map((action) => ({
+      $id: action.payload.doc.id, ...action.payload.doc.data() as Doctor,
+    })));
+  }
+
+  public getDoctor() {
```

Comment:
**Suggestion:** Change name to `getDoctors` as it returns multiple doctors.

Reply:
Fixed.